aiida/engine/processes/workchains/restart.py

```diff
@@ -295,6 +295,10 @@ def inspect_process(self) -> Optional['ExitCode']:  # pylint: disable=too-many-b
 
         return None
 
+    def get_outputs(self, node) -> Mapping[str, orm.Node]:
+        """Return a mapping of the outputs that should be attached as outputs to the work chain."""
```

Comment:
Should we still add a sentence of two explaining that this method can be overridden in case the `BaseRestartWorkChain` needs to adjust the outputs of the underlying `CalcJob` somehow?